tests/manifests/test-cluster-on-pvc-encrypted.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+  name: rook-ceph
+  namespace: rook-ceph
+spec:
+  dataDirHostPath: /var/lib/rook
+  mon:
+    count: 1
+    volumeClaimTemplate:
+      spec:
+        storageClassName: manual
+        resources:
+          requests:
+            storage: 5Gi
+  cephVersion:
+    image: ceph/daemon-base:latest-nautilus-devel
```

Comment:
Remind me, what is it we need from `latest-nautilus-devel`? Are we blocked on supporting this feature until the next ceph patch release is out?

Reply:
It is needed to get the raw encryption support from ceph-volume.